### ISSUE TITLE
gpaste-reloaded@feuerfuchs.eu applet update for cinnamon 6.4

### ISSUE THREAD

**Issue by fredcw** [fredcw/Notes#12]
@alb-soul

>Gpaste applet by feuerfuchs stops working in cinnamon 6.4.0...

Here's an updated version of the applet for cinnamon 6.4:

[gpaste-reloaded.zip](https://github.com/user-attachments/files/18330475/gpaste-reloaded.zip) (updated)

**Comment by alb-soul** [comment]
> @alb-soul
> 
> > Gpaste applet by feuerfuchs stops working in cinnamon 6.4.0...
> 
> Here's an updated version of the applet for cinnamon 6.4:
> 
> [gpaste-reloaded.zip](https://github.com/user-attachments/files/17965572/gpaste-reloaded.zip)

Works like a charm! Thanks 👍

![gpaste-reloaded-solved](https://github.com/user-attachments/assets/17f42fe1-27e2-4445-8db2-bcc0796e3fd0)

Will this fix be released to the Cinnamon applet, so we can download it directly from there without manually installing from a zip file?

**Comment by dosmanak** [comment]
Hi, thank you for the fix, altough. I hesitate to add it as zip file. Is there a PR open for this fix?

**Comment by fredcw** [comment]
@benmpl
>Refactoring for compatibility with the dialog changes, otherwhise it would break.

Actually, this needs updating again because of the new changes to ModalDialog - clicking on "+New item" more than once causes a cinnamon crash. I'll update it ASAP.

**Comment by fredcw** [comment]
@benmpl
in file `gpaste-reloaded@feuerfuchs.eu/6.4/GPasteNewItemDialog.js` line 13, replace:
```
super._init({ styleClass: 'gpaste__new-item-dialog' });
```
with:
```
super({ styleClass: 'gpaste__new-item-dialog', destroyOnClose: false });
```

**Comment by rubenpedev** [comment]
@fredcw 
> @benmpl in file `gpaste-reloaded@feuerfuchs.eu/6.4/GPasteNewItemDialog.js` line 13, replace:

Thanks, and I'm also including that in GPasteNotInstalledDialog

**Comment by fredcw** [comment]
I've updated the zip file above to include a fix for the "+New item" dialog bug.

**Comment by fredcw** [comment]
@benmpl

>Thanks, and I'm also including that in GPasteNotInstalledDialog

Yes, I just noticed that myself as well. And added it to the new .zip above. :)

**Comment by fredcw** [comment]
This applet has been updated in the Mint repos, so can be closed.
